### DISCUSSION TITLE
Error: Feature names of counts matrix cannot be empty

### DISCUSSION THREAD

**Discussion by mcsintonn** [satijalab/seurat#5903] [General]
**Moved this to issues**

Hi,

I'm brand new to single cell and I'm trying to use Seurat for the first time. I have a CellRanger output, with a raw_feature_bc_matrix folder. This folder contains three compressed files: barcodes.tsv.gz, features.tsv.gz, and matrix.mtx.gz. 

I have read this folder into R Studio using the following line, as recommended in the Seurat guide:

**naive1.data <- Read10X(data.dir = "/Users/Desktop/naive_raw_feature_bc_matrix")**

I then try to use the CreateSeuratObject command as follows:

**adipnaive <- CreateSeuratObject(counts = naive1.data, project = "TrypWAT", min.cells = 3, min.features = 200)**

At this point, I get the following error: 
**Error: Feature names of counts matrix cannot be empty**

I'm completely lost here, and was wondering if anyone has any insights into what might be causing the problem?  I was wondering if it might be because as well as mouse genes, there are also trypanosome genes in this file. I have attached an example of the file (although in Excel format rather than .tsv) 
[features_file_example.xlsx](https://github.com/satijalab/seurat/files/8602343/features_file_example.xlsx)

Any advice would be very much appreciated!

**Comment by howtofindme** [comment]
> **Moved this to issues**
> 
> Hi,
> 
> I'm brand new to single cell and I'm trying to use Seurat for the first time. I have a CellRanger output, with a raw_feature_bc_matrix folder. This folder contains three compressed files: barcodes.tsv.gz, features.tsv.gz, and matrix.mtx.gz.
> 
> I have read this folder into R Studio using the following line, as recommended in the Seurat guide:
> 
> **naive1.data <- Read10X(data.dir = "/Users/Desktop/naive_raw_feature_bc_matrix")**
> 
> I then try to use the CreateSeuratObject command as follows:
> 
> **adipnaive <- CreateSeuratObject(counts = naive1.data, project = "TrypWAT", min.cells = 3, min.features = 200)**
> 
> At this point, I get the following error: **Error: Feature names of counts matrix cannot be empty**
> 
> I'm completely lost here, and was wondering if anyone has any insights into what might be causing the problem? I was wondering if it might be because as well as mouse genes, there are also trypanosome genes in this file. I have attached an example of the file (although in Excel format rather than .tsv) [features_file_example.xlsx](https://github.com/satijalab/seurat/files/8602343/features_file_example.xlsx)
> 
> Any advice would be very much appreciated!

same problem ,have you solved it?

**Comment by qingmemng** [comment]
Hi
I meet the same problem?
